antsibull/data/sphinx_init/conf_py.j2

```diff
@@ -0,0 +1,31 @@
+# This file only contains a selection of the most common options. For a full list see the
+# documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+project = 'Ansible collections'
+copyright = 'Ansible contributors'
+
+html_short_title = 'Ansible Collections Documentation'
+
+extensions = ['sphinx.ext.autodoc', 'sphinx.ext.intersphinx', 'sphinx_antsibull_ext']
+
+pygments_style = 'ansible'
+
+highlight_language = 'YAML+Jinja'
+
+html_theme = 'sphinx_rtd_theme'
+html_show_sphinx = False
+
+display_version = False
+
+html_use_smartypants = True
+html_use_modindex = False
+html_use_index = False
+html_copy_source = False
+
+intersphinx_mapping = {
+    'python': ('https://docs.python.org/2/', (None, '../python2.inv')),
+    'python3': ('https://docs.python.org/3/', (None, '../python3.inv')),
+    'jinja2': ('http://jinja.palletsprojects.com/', (None, '../jinja2.inv')),
+    'ansible_4': ('https://docs.ansible.com/ansible/4/', (None, '../ansible_4.inv')),
```

Comment:
Why does ansible_4 have an underscore and python3 doesn't? A little consistency here wouldn't hurt.

Reply:
That's the traditional Ansible way of writing these (https://github.com/ansible/ansible/blob/devel/docs/docsite/sphinx_conf/ansible_conf.py#L296).

Reply:
Yes, and it's inconsistent. You have a chance to fix it on scale here.

Reply:
True :) Fixed in 5e64a61ec74c872835dab9dc48431f929fc6dd2a.